### ISSUE TITLE
don't use a share-alike license, this prevents re-use

### ISSUE THREAD

**Issue by LinguList** [Verkerk/DravLex#1]
Use the most generous CC license, without any restrictions, as this is our common policy (and helps us to reuse other peoples data as well).

**Comment by xrotwang** [comment]
Yes, as explained [here](https://wiki.creativecommons.org/wiki/ShareAlike_interpretation) some publications won't allow share-alike-licensed content, e.g. PLOS ONE.
